test/command/suite/select/filter/near_phrase/min_interval/with_max_element_intervals/match.expected

```diff
@@ -0,0 +1,42 @@
+table_create Entries TABLE_NO_KEY
+[[0,0.0,0.0],true]
+column_create Entries content COLUMN_SCALAR Text
+[[0,0.0,0.0],true]
+table_create Terms TABLE_PAT_KEY ShortText   --default_tokenizer 'TokenNgram("unify_alphabet", false,                                   "unify_digit", false)'   --normalizer NormalizerNFKC121
+[[0,0.0,0.0],true]
+column_create Terms entries_content COLUMN_INDEX|WITH_POSITION Entries content
+[[0,0.0,0.0],true]
+load --table Entries
+[
+{"content": "abc123456789def"}
+]
+[[0,0.0,0.0],1]
+select Entries   --filter 'content *NP-1,0,12,11"abc def"'   --output_columns '_score, content'
```

Comment:
Currently, when `max_elements_intervals` specified, the interval between `abc` and `def` for `abc123456789def` is regarded as:

`11` when checking `min_interval`.
`12` when checking `max_interval`.

This is because when checking `max_interval` and `min_interval`, the interval is calculated as `11`, and when checking `max_element_intervals` and `min_interval`, the interval is calculated as `12`.

I think this is a bug and I plan to work to fix this #1519.